### PR TITLE
Make bridged interactors unbuffered by default

### PR DESCRIPTION
**Pull request by kiritofeng**

### CODE REVIEW

File: testsuite/bridged_interactor_noflush/init.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+unbuffered: True
```
Comment (Xyene) [inline testsuite/bridged_interactor_noflush/init.yml:1]: BRO

**Review by Xyene — CHANGES_REQUESTED** [review]

File: testsuite/bridged_interactor_noflush/init.yml (hunk 1 above)
Comment (kiritofeng) [reply testsuite/bridged_interactor_noflush/init.yml:1]: if it make you feel worse, there's one that says `unbuffered: yes`

Comment (Xyene) [reply testsuite/bridged_interactor_noflush/init.yml:1]: It doesn't, because `yes` is actually part of the YAML spec.

Comment (kiritofeng) [reply testsuite/bridged_interactor_noflush/init.yml:1]: big TIL moment

**Review by Xyene — APPROVED** [review]

**Pull request merged**